apps/site/assets/css/_amenity.scss

```diff
@@ -15,3 +15,47 @@
     margin: .65rem;
   }
 }
+
+.access-amenities-header {
+  background: $brand-primary-darkest;
+  color: $white;
+  height: 3.75rem;
+}
+
+tr:nth-child(2n+3) {
+  background-color: $gray-lightest;
+}
+
+.access-amenities-row {
+  height: 5rem;
+}
+
+.status {
+  padding-right: 3.125rem;
+}
+
+.access-amenities-name {
+  padding-left: 1rem;
+}
+
+.access-amenities-text {
+  font-size: .875rem;
+  font-weight: 400;
+}
```

Comment:
I am curious about the purpose of the `font-weight: 400`.  According to Mozilla a weight of 400 is regular text weight.  I don't see what could be affecting this font weight in the layout below.  I am wondering if the `font-weight` is needed here.

non-blocking: the `font-size: .875rem` already exists as a helper class in `_typography.scss` (`fs-14`)